### PR TITLE
ReceivedCharacter.char

### PR DESCRIPTION
**Pull request by tomara-x**
i hope this is okay

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: content/learn/migration-guides/0.12-to-0.13.md [hunk 1]
```diff
@@ -1483,6 +1483,14 @@ Consider changing usage:
 
 `Window` has a new [`name`](https://docs.rs/bevy/latest/bevy/prelude/struct.Window.html#structfield.name) field for specifying the "window class name." If you don't need this, set it to `None`.
 
+### [ReceivedCharacter.char is now SmolStr instead of char]
+
+<div class="migration-guide-area-tags">
+    <div class="migration-guide-area-tag">Windowing</div>
+</div>
+
+Use `.chars().nth(0)`
```
Comment (alice-i-cecile) [inline content/learn/migration-guides/0.12-to-0.13.md:1492]:
```suggestion
To restore the previous behavior, use `.chars().nth(0)`
```

Comment (alice-i-cecile) [reply content/learn/migration-guides/0.12-to-0.13.md:1492]: I think there are some cases where these can actually be multiple characters long with unusual keyboard layouts.

Comment (tomara-x) [reply content/learn/migration-guides/0.12-to-0.13.md:1492]: yeah, that's better :)

Comment (tomara-x) [reply content/learn/migration-guides/0.12-to-0.13.md]: > I think there are some cases where these can actually be multiple characters long with unusual keyboard layouts.

whoa! cool!

**Review by TrialDragon — APPROVED** [review]

**Pull request merged**